org.twinery.Twine.svg

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
why not using the svg in the repository?

Reply:
And it's removed. Good catch!